doc/changelog/07-vernac-commands-and-options/15139-search-in.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  :cmd:`Search` understands modifier ``in`` as an alias of ``inside``
```

Comment:
```suggestion
  :cmd:`Search` now accepts ``in`` as an alias of ``inside``
```

Reply:
Or maybe say `Search*` without the `:cmd:` since 3 commands were changed.

Reply:
Or just list the three commands.